### ISSUE TITLE
[Feature Request]: Sidebar fails color contrast accessability standards

### ISSUE THREAD

**Issue by jordanlambrecht** [WDaan/VueTorrent#2609]
### Description

I'd like to make a case for changing the colors on the sidebar. 

The green background gives the false indication that the entire sidebar is "selected" from a UI psychology standpoint. Additionally, the sidebar is a secondary object and the bright green background immediately draws the user's focus away from the primary content.

Most importantly, none of the color combinations fully pass modern color contrast accessibility standards:

<img width="1508" height="798" alt="Image" src="https://github.com/user-attachments/assets/9454f6ac-75d0-4df7-8fcf-5a64b4659e37" />
<img width="1488" height="792" alt="Image" src="https://github.com/user-attachments/assets/7f479ddc-6176-4221-aabc-adb4a053ff18" />
<img width="1560" height="490" alt="Image" src="https://github.com/user-attachments/assets/b67abcd8-9079-41de-9eb0-513ae0d4de30" />
<img width="1478" height="764" alt="Image" src="https://github.com/user-attachments/assets/7165cc68-bb57-433d-a2a4-3ba2b6516468" />
<img width="1474" height="786" alt="Image" src="https://github.com/user-attachments/assets/4e2193ce-e5b4-4c09-b27e-8167cb1caedb" />

### Describe the solution you'd like

I think Theme Park's Dracula pack does a great job of handling color for the sidebar:

<img width="3450" height="1928" alt="Image" src="https://github.com/user-attachments/assets/9dbb04c1-ca4a-40e7-b249-a85ba1a17505" />

### Complementary informations

- [ ] Is this feature already implemented in the default WebUI?
